package.json

```diff
@@ -10,6 +10,7 @@
     "npm-css": "https://registry.npmjs.org/npm-css/-/npm-css-0.2.3.tgz",
     "postcss-discard-comments": "https://registry.npmjs.org/postcss-discard-comments/-/postcss-discard-comments-2.0.4.tgz",
     "postcss-filter-plugins": "https://registry.npmjs.org/postcss-filter-plugins/-/postcss-filter-plugins-2.0.2.tgz",
+    "sanitize-html": "^1.26.0",
```

Comment:
This package looks to be a large size, could also be adding to the SDK size
https://www.npmjs.com/package/sanitize-html

Reply:
Yeah I'll have to figure out a better way to clean inputs